robocop/checkers/spacing.py

```diff
@@ -0,0 +1,119 @@
+"""
+Spacing checkers
+"""
+from robot.parsing.model.blocks import TestCase, Keyword
+from robot.parsing.model.statements import EmptyLine, Comment
+from robocop.checkers import RawFileChecker, VisitorChecker
+from robocop.rules import RuleSeverity
+
+
+def register(linter):
+    linter.register_checker(InvalidSpacingChecker(linter))
+    linter.register_checker(MissingTrailingBlankLineChecker(linter))
+
+
+class InvalidSpacingChecker(RawFileChecker):
+    """ Checker for invalid spacing. """
+    rules = {
+        "1001": (
+            "trailing-whitespace",
+            "Trailing whitespace at the end of line",
+            RuleSeverity.WARNING
+        )
+    }
+
+    def check_line(self, line, lineno):
+        stripped_line = line.rstrip('\n')
+        if stripped_line:
+            if stripped_line[-1] == ' ':
+                self.report("trailing-whitespace", lineno=lineno, col=len(stripped_line))
+
+
+class MissingTrailingBlankLineChecker(VisitorChecker):
+    """ Checker for invalid spacing. """
+    rules = {
+        "1002": (
+            "missing-trailing-blank-line",
```

Comment:
is this checked somewhere?

Reply:
This is nit implemented yet. I can remove it and will do it in another PR.